pkg/services/searchV2/index.go

```diff
@@ -421,6 +473,11 @@ func (i *searchIndex) buildOrgIndex(ctx context.Context, orgID int64) (int, erro
 	i.perOrgIndex[orgID] = index
 	i.mu.Unlock()
 
+	time.Sleep(20 * time.Second)
```

Comment:
I'll approve assuming this gets removed 🤣

Reply:
🤦